backend/src/api/bitcoin/bitcoin-client.ts

```diff
@@ -2,12 +2,15 @@ import config from '../../config';
 const bitcoin = require('../../rpc-api/index');
 import { BitcoinRpcCredentials } from './bitcoin-api-abstract-factory';
 
+export const defaultCookiePath = `${process.env.HOME}/.bitcoin/${{mainnet:'',testnet:'testnet3/',signet:'signet/'}[config.MEMPOOL.NETWORK]}.cookie`;
+
```

Comment:
I don't think we want to do this, you can configure your cookie path in the configuration file and the default should be `/bitcoin/.cookie`

Reply:
this is the cookie location on my device, i run bitcoind and mempool from the same unix account without docker

on which systems is `/bitcoin` the bitcoind data dir?

Reply:
the official mempool.space servers

Reply:
ok, changing this